.github/workflows/test.yml

```diff
@@ -45,31 +45,31 @@ jobs:
     - name: Full integration test
       run: |
         ./runkmd.sh \
-            ci/travis/${RELEASE} \
-            ci/travis/repository.yml \
+            ci/${RELEASE} \
+            ci/repository.yml \
             --workspace /tmp/kmd-ws \
             --cache /tmp/kmd-cache \
             --prefix /tmp/pfx \
-            --release travis \
-            --locations-config $(realpath ci/travis/locations.yml) \
+            --release ci \
+            --locations-config $(realpath ci/locations.yml) \
 
     - name: Build libkmd.so for testing
```

Comment:
We might add "(a dummy C library)" to avoid people believing `libkmd` is something useful..